widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/ViewExstensions.kt

```diff
@@ -320,3 +323,12 @@ internal fun GliaSurveyOptionButton.applyOptionButtonTheme(theme: OptionButtonTh
         )
     )
 }
+
+internal fun Activity.prepareContextWithStyle() : Context {
+    return MaterialThemeOverlay.wrap(
+        this,
+        null,
+        R.attr.gliaChatStyle,
+        R.style.Application_Glia_Chat
+    )
+}
```

Comment:
If we are extracting this to an extension function, I think it makes sense to use it in other places where `MaterialThemeOverlay.wrap()` is called. What do you think?

Reply:
could be better to rename to `wrapWithMaterialThemeOverlay` to indicate what is this function doing, also I think we should have parameters for this method with default values to be able to use from all views.

Reply:
done

Reply:
And most places we are using this from is in constructor, so I don't think we can universally use it

Reply:
Sorry for the late addition, maybe it makes sense, in this case, to create this method on Context instance like - `Context.wrapWithMaterialThemeOverlay(...)`

Reply:
done